source/extensions/extensions_build_config.bzl

```diff
@@ -172,4 +172,9 @@ EXTENSIONS = {
     #
 
     "envoy.filters.http.cache.simple_http_cache":       "//source/extensions/filters/http/cache/simple_http_cache:simple_http_cache_lib",
+
+    # Internal redirect predicates
+    #
```

Comment:
optional: consider a # ahead of internal redirect predicates line, to match with prior sections

Reply:
Done.